runtime/main/src/main/java/org/qbicc/runtime/main/VMHelpers.java

```diff
@@ -632,4 +618,12 @@ public static void JLT_start0(void_ptr_unaryoperator_function_ptr runFuncPtr, vo
             throw new InternalError("pthread error code: " + result);
         }
     }
+
+    // TODO: Class library compatibility kludge; remove once qbicc 0.3.0 is released
```

Comment:
Poking at this a bit more to see if I can avoid this or reduce it....not 100% sure why this is needed, but I needed at least some bits of it to get Thread.start to compile correctly.

Reply:
we need to leave a forwarding method for threadWrapper. Reduced to minimal kludge and left a comment about what to update post qbicc-rt 0.3.0